mods/cnc/rules/aircraft.yaml

```diff
@@ -138,17 +139,17 @@ ORCA:
 	Armament@PRIMARY:
 		Weapon: OrcaAGMissiles
 		LocalOffset: 427,-171,-213, 427,171,-213
+		PauseOnCondition: ammo < 1
```

Comment:
Most other aircraft use `!ammo`. Can you please make these consistent?

Reply:
I just overlooked this one, fixed.